addons/mail/static/src/js/chatter.js

```diff
@@ -98,8 +98,24 @@ var Chatter = Widget.extend({
         var def = this._dp.add(Promise.all(fieldDefs));
         this._render(def).then(this._updateMentionSuggestions.bind(this));
 
+        // Drag-Drop files
+        // Allowing body to detect dragenter and dragleave for display
+        var $body = $('body');
+        $body.on('dragleave', this._onBodyFileDragLeave.bind(this));
+        $body.on("dragover", this._onBodyFileDragover.bind(this));
+        $body.on("drop", this._onBodyFileDrop.bind(this));
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    destroy() {
+        var $body = $('body');
+        $body.on('dragleave');
```

Comment:
should be `.off()`, you may need to name these events with `.<name>` (works only with jQuery events)